src/instance.rs

```diff
@@ -221,13 +221,26 @@ unsafe impl<T> AsPyPointer for Py2<'_, T> {
 ///
 /// The advantage of this over `&Py2` is that it avoids the need to have a pointer-to-pointer, as Py2
 /// is already a pointer to an `ffi::PyObject``.
+///
+/// Similarly, this type is `Copy` and `Clone`, like any pointer.
```

Comment:
I think this should be "like shared references". Because it does not act as a raw pointer and mutable references are not `Copy`.

Reply:
Yep agreed 👍